go.mod

```diff
@@ -3,6 +3,6 @@ module github.com/sourcegraph/deploy-sourcegraph-docker
 go 1.15
 
 require (
-	github.com/sourcegraph/sourcegraph/enterprise/dev/ci/images v0.0.0-20220511200824-abaeea3fde03
+	github.com/sourcegraph/sourcegraph/enterprise/dev/ci/images v0.0.0-20220922183713-4ab710dc0025
```

Comment:
Did you mean to include this? I think this is the old pin?

Reply:
I'm confused. I changed it to the new pin by cherry-picking your commit from `master`, right? This updates it from a version from May to a version from September

Reply:
🤦 sorry, misread the diff